src/typesAndConstants/AbilityJSON.tsx

```diff
@@ -0,0 +1,8 @@
+type AbilityJSON = {
```

Comment:
typesAndConstants is bad name. if you have `and` in the name its a bad sign for the name.
have different folders